File: ios/Sources/ScanditIdVoidedDetectionNativePlugin/ScanditIdVoidedDetectionNativePlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(ScanditIdVoidedDetectionNativePlugin)
public class ScanditIdVoidedDetectionNativePlugin: CAPPlugin, CAPBridgedPlugin {
    public let identifier = "ScanditIdVoidedDetectionNativePlugin" 
    public let jsName = "ScanditIdVoidedDetectionNative" 
    public let pluginMethods: [CAPPluginMethod] = [
        CAPPluginMethod(name: "echo", returnType: CAPPluginReturnPromise),
    ] 
    @objc func echo(_ call: CAPPluginCall) {
        call.resolve()
    }
}

File: android/src/main/java/com/scandit/capacitor/datacapture/id_voided_detection/ScanditIdVoidedDetectionNativePlugin.java
package com.scandit.capacitor.datacapture.id_voided_detection;

import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "ScanditIdVoidedDetectionNative")
public class ScanditIdVoidedDetectionNativePlugin extends Plugin {
  @PluginMethod
  public void echo(PluginCall call) {
    call.resolve();
  }
}
